.changeset/slow-timers-enjoy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': patch
```

Comment:
```suggestion
'@shopify/polaris': major
```

Should this be a major change?

Reply:
Yes, thanks for the callout. To avoid (unintentionally) releasing a major version with `/snapit`, we plan to merge feature branches with `patch` and update changesets to `major` at the end.